### DISCUSSION TITLE
Welcome to Obtainium-Emulation-Pack Discussions!

### DISCUSSION THREAD

**Discussion by RJNY** [RJNY/Obtainium-Emulation-Pack#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other Obtainium Emulation Pack Users. This is a forum for:
  * Questions you’re wondering about.
  * Sharing ideas.
  * Feature Requests

**Comment by 7ank0v1c** [comment]
Heya, love this, it really helped me a lot with my set-up. I was just wondering if maybe you was able to add DuckStation standalone (beta preferably). I believe this might be helpful for others and myself in the future. Here is the code I use currently 

`        {
            "id": "com.github.stenzek.duckstation",
            "url": "https://www.duckstation.org/android/",
            "author": "www.duckstation.org",
            "name": "DuckStation",
            "installedVersion": "0.1-8108",
            "latestVersion": "0.1-8108",
            "apkUrls": "[[\"1069342510-duckstation-beta.apk\",\"https://www.duckstation.org/android/duckstation-beta.apk\"]]",
            "otherAssetUrls": "[]",
            "preferredApkIndex": 0,
            "additionalSettings": "{\"intermediateLink\":[],\"customLinkFilterRegex\":\"https://www.duckstation.org/android/duckstation-beta.apk\",\"filterByLinkText\":false,\"skipSort\":false,\"reverseSort\":false,\"sortByLastLinkSegment\":false,\"versionExtractWholePage\":true,\"requestHeader\":[{\"requestHeader\":\"User-Agent: Mozilla/5.0 (Linux; Android 10; K) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/114.0.0.0 Mobile Safari/537.36\"}],\"defaultPseudoVersioningMethod\":\"APKLinkHash\",\"trackOnly\":false,\"versionExtractionRegEx\":\"(\\\\d.\\\\d-\\\\d+).*\",\"matchGroupToUse\":\"$1\",\"versionDetection\":false,\"useVersionCodeAsOSVersion\":false,\"apkFilterRegEx\":\"\",\"invertAPKFilter\":false,\"autoApkFilterByArch\":true,\"appName\":\"\",\"shizukuPretendToBeGooglePlay\":false,\"allowInsecure\":false,\"exemptFromBackgroundUpdates\":false,\"skipUpdateNotifications\":false,\"about\":\"\",\"refreshBeforeDownload\":false,\"supportFixedAPKURL\":false}",
            "lastUpdateCheck": 1733920860647818,
            "pinned": false,
            "categories": [
                "Emulator"
            ],
            "releaseDate": null,
            "changeLog": null,
            "overrideSource": null,
            "allowIdChange": false
        },`

**Reply by RJNY** [reply]
I think that's a great suggestion and I'll add it in soon. Thanks for raising this suggestion.
I'm leaning towards adding the stable version, as most of the people that use this are _just following some guide on the internet_ and are complete beginners. It's best if things just work.

In the meantime, are you familiar with the obtainium creators crowdsource page? I have contributed to it myself. They also have the stable & beta releases for duck station.
https://apps.obtainium.imranr.dev/

**Reply by RJNY** [reply]
Actually, I just had a "hold my beer" moment. Went ahead and cut a release with both beta and stable.
